src/Infra/Communication/ApplicationMananger.cs

```diff
@@ -4,167 +4,199 @@
 using Microsoft.Windows.Apps.Test.Foundation;
 using Microsoft.Windows.Apps.Test.Foundation.Waiters;
 using System;
-using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
+using WinAppDriver.Infra.Helper;
+using WinAppDriver.Infra.ModernApp;
 
 namespace WinAppDriver.Infra.Communication
 {
-  public class ApplicationMananger : IApplicationManager
-  {
-    private UIObject GetTopLevelWindow(UIObject uiObject)
+    public class ApplicationMananger : IApplicationManager
```

Comment:
Thanks to contribute this. Can you reformat all the files. I guess I convert `tab` to 2 spaces while you used 4 spaces